.env.dist

```
@@ -0,0 +1,45 @@
+DOCKER_ECR=asd.dkr.ecr.ap-southeast-2.amazonaws.com/
```

```suggestion
DOCKER_ECR=<account-id>.dkr.ecr.<region>.amazonaws.com/
```

same as in FM for example: https://github.com/ausaccessfed/federationmanager/blob/608449e2a50c04acda820cccd7d64c956e99788a/env.dist#L7